services/121-service/test/registrations/bulk-update-registration.test.ts

```diff
@@ -12,6 +11,22 @@ import {
   resetDB,
 } from '@121-service/test/helpers/utility.helper';
 
+function filterUnchangedProperties(
```

Comment:
I added this function at the top of the test file. I am not sure if it should move to one of the helper files, or what our practice should be here. I expect that this helper function will only ever be use in this files, which argues for keeping it here, but we usually do not have seperate fuctions inside .test.ts files